Assets/HoloToolkit/Input/Scripts/KeywordManager.cs

```diff
@@ -2,79 +2,97 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using System.Collections.Generic;
-using System.Linq;
 using UnityEngine;
-using UnityEngine.Events;
 using UnityEngine.Windows.Speech;
 
 namespace HoloToolkit.Unity
 {
     /// <summary>
-    /// KeywordManager allows you to specify keywords and methods in the Unity
+    /// KeywordManager allows you to specify keywords in the Unity
     /// Inspector, instead of registering them explicitly in code.
     /// This also includes a setting to either automatically start the
     /// keyword recognizer or allow your code to start it.
     ///
     /// IMPORTANT: Please make sure to add the microphone capability in your app, in Unity under
     /// Edit -> Project Settings -> Player -> Settings for Windows Store -> Publishing Settings -> Capabilities
     /// or in your Visual Studio Package.appxmanifest capabilities.
+    /// 
+    /// Please, also make sure to also set your keyword responses in the Inspector window under Keywords and Responses Array.
     /// </summary>
-    public partial class KeywordManager : MonoBehaviour
+    public partial class KeywordManager : Singleton<KeywordManager>
     {
+        /// <summary>
+        /// Occurs when a registered keyword is spoken.
+        /// </summary>
+        /// <param name="keyword">Keyword spoken.</param>
+        public delegate void OnKeywordRecognizedEvent(string keyword);
+        public event OnKeywordRecognizedEvent OnKeywordRecognized;
```

Comment:
I think we already discussed finding a way to keep the functionality of assigning Events in the Inspector elsewhere.